### DISCUSSION TITLE
[求助]umijs/max v4.0.17 编译报错Variable @screen-xs is undefined

### DISCUSSION THREAD

**Discussion by ByYogi** [umijs/umi#11632] [Q&A] [answered]
```json
"dependencies": {
    "@ant-design/happy-work-theme": "^1.0.0",
    "@ant-design/icons": "^5.2.6",
    "@ant-design/pro-components": "2.4.4",
    "@umijs/max": "^4.0.17",
    "antd": "5.8.6",
    "dayjs": "^1.11.9",
    "js-cookie": "^3.0.1",
    "lodash": "^4.17.21",
  },
```
![image](https://github.com/umijs/umi/assets/111735598/8a4cb1bf-c406-44b8-982e-f9fd9df15fd3)

**Comment by fz6m** [comment, marked as answer]
升级到最新的 `@umijs/max` 版本看看，如果还有问题，需要给一个最小复现。

**Reply by ByYogi** [reply]
因为新版`umijs`与AntProComp有冲突bug。所以特意降低版本。但是回不去了（哭）
这个issue https://github.com/ant-design/pro-components/issues/7624#issuecomment-1704812095

**Reply by fz6m** [reply]
如果你显示安装了某个版本的 pro component ，就会使用你安装的这个版本，所以你需要哪个特定版本自己安装就好了，如果新版 pro component 有问题，可以通过给 pro component PR 解决，和 umi 版本无关，umi 不会影响你描述的这个表单。
